src/views/SpaceProposal.vue

```diff
@@ -266,7 +284,44 @@ onMounted(async () => {
               </div>
             </UiDropdown>
           </div>
-          <UiMarkdown :body="proposal.body" class="mb-6" />
+          <div class="relative">
```

Comment:
How about making this feature a component, so you move the logic on another page and make it reusable on others places

Reply:
Hm yeah thought about that but couldn't think of another use-case on our Ui. Also it's pretty specific to this page.